src/main/java/com/cleancommunity/user/UserController.java

```diff
@@ -6,49 +6,45 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.util.ArrayList;
 import java.util.List;
 
 @RestController
 public class UserController {
 
-//    private com.cleancommunity.com.cleancommunity.user.UserDAO userDAO = new com.cleancommunity.com.cleancommunity.user.UserDAO();
-//    private com.cleancommunity.com.cleancommunity.user.UserFactory userFactory = new com.cleancommunity.com.cleancommunity.user.UserFactory();
+    private UserDAO userDAO = new MysqlUserDAO();
+    private UserFactory userFactory = new UserFactory();
 
     @RequestMapping(value = "/users/", method = RequestMethod.POST)
     public Response newUser(@RequestParam(value="request") String request) {
-        // create new com.cleancommunity.user
-//        com.cleancommunity.com.cleancommunity.user.com.cleancommunity.com.cleancommunity.user newUser = userFactory.getNewUserByRequest(request);
-//        if (!userDAO.addUser(newUser)) {
-//            com.cleancommunity.misc.Response.getServerErrorResponse();
-//        }
+        // create new user
+        User newUser = userFactory.createUserByRequest(request);
+        if (!userDAO.addUser(newUser)) {
+            com.cleancommunity.misc.Response.getServerErrorResponse();
+        }
 
         return Response.getSuccessResponse();
     }
 
     @RequestMapping(value = "/users/", method = RequestMethod.GET)
     public List<User> getAllUsers(@RequestParam(value="request") String request) {
         // get all users
-//        return userDAO.getUsers();
-        return new ArrayList<>();
+        return userDAO.getUsers();
```

Comment:
Similar to PostingController, does this need to be formatted in any way? Or will this be done in a higher-level class?

Reply:
Spring will take care of this for us. My guess is it will make it a json array.